Form_1/web_form.html

```diff
@@ -0,0 +1,104 @@
+            <fieldset>
+              <legend>Title</legend>
+              <ul>
+                  <li>
+                    <label for="title_1">
+                      <input type="radio" id="title_1" name="title" value="K">
+                      King
+                    </label>
+                  </li>
+                  <li>
+                    <label for="title_2">
+                      <input type="radio" id="title_2" name="title" value="Q">
+                      Queen
+                    </label>
+                  </li>
+                  <li>
+                    <label for="title_3">
+                      <input type="radio" id="title_3" name="title" value="J">
+                      Joker
+                    </label>
+                  </li>
+              </ul>
+            </fieldset>
+            <p>
+              <label for="name">
+                <span>Name: </span>
+                <strong><span aria-label="required">*</span></strong>
+              </label>
+              <input type="text" id="name" name="username">
+            </p>
+            <p>
+              <label for="mail">
+                <span>E-mail: </span>
+                <strong><span aria-label="required">*</span></strong>
+              </label>
+              <input type="email" id="mail" name="usermail">
+            </p>
+            <p>
+              <label for="pwd">
+                <span>Password: </span>
+                <strong><span aria-label="required">*</span></strong>
+              </label>
+              <input type="password" id="pwd" name="password">
+            </p>
+        </section>
+        
+        <!--Second Section-->
+        <section>
+            <h2>Payment information</h2>
+            <p>
+              <label for="card">
+                <span>Card type:</span>
+              </label>
+              <select id="card" name="usercard">
+                <option value="visa">Visa</option>
+                <option value="mc">Mastercard</option>
+                <option value="amex">American Express</option>
+              </select>
+            </p>
+            <p>
+              <label for="number">
+                <span>Card number:</span>
+                <strong><span aria-label="required">*</span></strong>
+              </label>
+              <input type="tel" id="number" name="cardnumber">
+            </p>
+            <p>
+              <label for="expiration">
+                <span>Expiration date:</span>
+                <strong><span aria-label="required">*</span></strong>
+              </label>
+              <input type="text" id="expiration" required="true" placeholder="MM/YY" pattern="^(0[1-9]|1[0-2])\/([0-9]{2})$">
+            </p>
+        </section>
+        
+        <!--Third Section -->
+        <section>
+            <p>
+              <button type="submit">Validate the payment</button>
+            </p>
+        </section>
+        
+    </form>
+</body>
+</html>
```

Comment:
Add a new line at the end of the file :) [Here is the reason ](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)why this should be